src/Repository/JobsRepository.php

```diff
@@ -16,27 +17,107 @@
  */
 class JobsRepository extends ServiceEntityRepository
 {
+    /**
+     * @param ManagerRegistry $registry
+     */
     public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Jobs::class);
     }
 
-    public function save(Jobs $entity, bool $flush = false): void
+    /**
+     * @param Jobs $entity
+     * @return void
+     */
+    public function save(Jobs $entity): void
     {
         $this->getEntityManager()->persist($entity);
-
-        if ($flush) {
-            $this->getEntityManager()->flush();
-        }
+        $this->getEntityManager()->flush();
     }
 
-    public function remove(Jobs $entity, bool $flush = false): void
+    /**
+     * @param Jobs $entity
+     * @return void
+     */
+    public function remove(Jobs $entity): void
     {
         $this->getEntityManager()->remove($entity);
+        $this->getEntityManager()->flush();
+    }
+
+    /**
+     * @param int $id
+     * @param array $params
+     * @return int
+     */
+    public function update(int $id, array $params): int
+    {
+        $queryBuilder = $this->createQueryBuilder('j');
+
+        $nbUpdatedRows = $queryBuilder->update()
```

Comment:
here are 2 issues:
1. priority and active are not managed for update.
2. need to check if 'name', 'description', 'company_id', 'active' and 'priority' are sent in the $params array. If one is missing we get an array key missing error.
the query builder needs to be constructed in steps like:

![image](https://user-images.githubusercontent.com/19302124/207572864-1a042796-ea46-4c44-82b5-86e4d609f258.png)